sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -183,10 +182,7 @@ def _open(self):
     def _send(self, message, timeout=None, last_exception=None):
         self._open()
         self._set_msg_timeout(timeout, last_exception)
-        try:
-            self._handler.send_message(message.message)
```

Comment:
I may revisit this comment as I read further, but I recall this being caught _very intentionally_ so that a user had a consistent MessageSendFailed exception to catch on regardless of what happened, if they needed a generic solution.

Reply:
I think ServiceBusError is that consistent exception to catch on regardless of what happened, except for those client-only validation error.